iphone/Classes/KrollObject.m

```diff
@@ -1347,7 +1347,17 @@ -(void)storeListener:(id)eventCallbackOrWrapper forEvent:(NSString *)eventName
 	{
 		TiValueRef jsCallbackArrayLength = TiObjectGetProperty(jsContext, jsCallbackArray, kTiStringLength, &exception);
 		int arrayLength = (int)TiValueToNumber(jsContext, jsCallbackArrayLength, &exception);
-		
+        
+		double lenght = TiValueToNumber(jsContext, jsCallbackArrayLength, NULL);
```

Comment:
isn't this the same as arrayLength that was previously declared? please explain why this has to be declared again?
and length has to be spelt correctly.